### ISSUE TITLE
How to move to the next month using button instead of Slide .....

### ISSUE THREAD

**Issue by GopinathKaliappan** [twinssbc/Ionic-Calendar#9]
How to move to the next month using button

<ion-slide-box class=\"monthview-slide\" on-slide-changed=\"slideChanged($index)\" show-pager=\"true\" does-continue=\"true\"\n" +

I'm Trying to use Button but dint working ...

 "<button type='button' class='btn btn-default btn-sm' ng-click=\"slideChanged($index+1)\" >Next

**Comment by Manduro** [comment]
As noted in the readme (https://github.com/twinssbc/Ionic-Calendar#events) you can broadcast these events to move to the previous or next view:

```js
// Move to the next view
$scope.$broadcast('changeDate', 1);

// Move to the previous view
$scope.$broadcast('changeDate', -1);
```

**Comment by twinssbc** [comment]
@GopinathKaliappan  Manduro is correct, feel free to have a try. Thanks.

**Comment by Manduro** [comment]
@twinssbc It is not implemented in this plugin, but in the normal Angular version. #11 has a fix.

**Comment by twinssbc** [comment]
@Manduro @GopinathKaliappan I have added missing event handler for changeDate event. It's included in version 0.1.3, feel free to have a try. Thanks.
